js/renovation/ui/editors/__tests__/check_box.test.tsx

```diff
@@ -366,6 +407,30 @@ describe('CheckBox', () => {
 
   describe('Logic', () => {
     describe('Getters', () => {
+      describe('icon styles', () => {
+        it('should have "width","height" styles', () => {
+          const checkBox = new CheckBox({ iconHeight: 22, iconWidth: 22 });
+
+          checkBox.updateIconFontSize();
+
+          expect(checkBox.iconStyles).toMatchObject({ width: '22px', height: '22px' });
+        });
+
+        each([22, '22px'])
+          .it('should convert %s" in "22px"', (value) => {
+            expect(new CheckBox({
+              iconHeight: value, iconWidth: value, width: 44, height: 44,
```

Comment:
do we really need `width` and `height` properties here?

Reply:
nope